core/init.c

```diff
@@ -3,7 +3,7 @@
 extern struct uwsgi_server uwsgi;
 
 struct http_status_codes {
-        const char      key[3];
+        const char      key[4];
```

Comment:
I don't see this from the issues mentioned above, is it for something else?

Reply:
I made an specific commit for this.